templates/include/glean/labeled_group.1.schema.json

```diff
@@ -2,5 +2,6 @@
 "propertyNames": {
   "type": "string",
   "pattern": "^[a-z_][a-z0-9_]{0,29}(\\.[a-z0-9_]{0,29})*$",
-  "maxLength": 30
+  "comment": "This must be at least the length of 'category.name' metric names to support error reporting",
+  "maxLength": 61
```

Comment:
Do we want to update the changelog with that? If so, let's also mention the related bug and the rationale.

Reply:
Good point.  I forgot we had a glean changelog over here.